src/Utilities/SystemsHelpers.js

```diff
@@ -49,16 +51,23 @@ export const buildActiveFiltersConfig = (filter, search, deleteFilters) => ({
 export const systemsColumnsMerger = (defaultColumns, additionalColumns) => {
     let lastSeen = defaultColumns.filter(({ key }) => key === 'updated');
     lastSeen = [{ ...lastSeen[0], key: 'last_upload', sortKey: 'last_upload' }];
+    const nameColumn = defaultColumns.filter(({ key }) => key === 'display_name');
+    const groupColumn = defaultColumns.filter(({ key }) => key === 'groups');
+    const tagsColumn = defaultColumns.filter(({ key }) => key === 'tags');
 
-    let name = defaultColumns.filter(({ key }) => key === 'display_name');
-    let tag = defaultColumns.filter(({ key }) => key === 'tags');
-
-    name = [{
-        ...name[0],
-        renderFunc: (displayName, id) => <InsightsLink to={`/systems/${id}`}>{displayName}</InsightsLink>
-    }];
+    groupColumn[0]?.renderFunc ? groupColumn[0].renderFunc = (data, ...value) => {
```

Comment:
According to my latest review of Georgii's [PR](https://github.com/RedHatInsights/insights-advisor-frontend/pull/1166/files#diff-fa905a9ffbddeb89214abf204bfdbdf2b254bd70fb83702b7d22c3adfeb5a7c6L16) for advisor, we do not have to tweak here anymore.

Reply:
True! now it works like a charm without these tricks!